packages/cli/src/models/Verifier.js

```diff
@@ -47,4 +52,103 @@ async function publishToEtherchain(params) {
   }
 }
 
+async function publishToEtherscan(params) {
+  if (params.apiKey === undefined) {
+    throw new Error('API key not specified.')
```

Comment:
Let's be a bit more explicit here, and rephrase the error to: "*Etherscan API key not specified. To get one, follow this link: https://etherscancom.freshdesk.com/support/solutions/articles/35000022163-i-need-an-api-key*", or something similar.